sql/monitoring/structured_detailed_error_counts_v1/view.sql

```diff
@@ -9,7 +9,7 @@ WITH error_examples AS (
     document_version,
     error_type,
     error_message,
-    `moz-fx-data-shared-prod.udf_js.gunzip`(ANY_VALUE(payload)) AS sample_payload,
+    `moz-fx-data-shared-prod.udf.js.gunzip`(ANY_VALUE(payload)) AS sample_payload,
```

Comment:
I thought it'd be best to use `<udf|udf_js>.<udf_name>` naming to distinguish persistent UDFs from temporary UDFs which are named something like `udf_<udf_name>`. However, it's easy to go back to the previous naming if that's better.

Reply:
Most of the changes that follow are renaming UDFs, changes from running the formatting script and deletions due to removing `templates/`

Reply:
I'm not understanding this comment. `moz-fx-data-shared-prod.udf.js.gunzip` looks like an invalid name containing 3 periods rather than 4. Is this a typo?

Reply:
Yeah, that was a typo